aiohttp/web_app.py

```diff
@@ -359,7 +359,7 @@ def __call__(self) -> "Application":
         return self
 
     def __repr__(self) -> str:
-        return "<Application 0x{:x}>".format(id(self))
+        return f"<Application 0x{id(self):x}>"
```

Comment:
Personally, I dislike the overuse of f-strings and think this is a little too aggressive. These changes are coming from pyupgrade.

Reply:
Please consider `pyupgrade` as yet another code formatter.
Say, sometimes `black` style is a little different than I used to write my personal code.
But I think it is ok: better to have an *uniform* style that is supported by automatic tooling than have no such tool.
Even if you prefer explicit `.format()` call f-strings are still readable and understandable I believe.